.env.vagrant

```diff
@@ -20,7 +20,7 @@ FILE_TRANSFER_FEATURE=1
 
 KAFKA_BROKER_ADDR=chs-kafka:9092
 
-INQUIRY_EMAIL='enquiries@companieshouse.gov.uk'
+ENQUIRY_EMAIL='enquiries@companieshouse.gov.uk'
```

Comment:
just looking at your config changes, we could have just reused the global environment property instead of creating a new one:
`ENQUIRIES_MAILBOX=mailto:enquiries@companieshouse.gov.uk`

Reply:
discussed - leave as is for now